<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinCourseCard</class>
 <widget class="QWidget" name="JoinCourseCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="join_course">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>500</width>
     <height>300</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>300</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华光超粗黑_CNKI</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    border-radius: 15px;
    padding-left: 10px;
    padding-right: 10px;
    color: rgb(255,255,255);
    min-width: 120px;
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(18,111,237), stop : 1 rgb(45,147,250));
}

QPushButton:hover {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(3,92,227), stop:1 rgb(21,133,244));
}

QPushButton:pressed {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(0,77,201), stop : 1 rgb(4,117,231));
}

QPushButton:disabled {
    background: qlineargradient(x1 : 0, x2 : 1, y1: 0, y2: 1, stop : 0 rgb(196,196,196), stop : 1 rgb(207,207,207));
}</string>
   </property>
   <property name="text">
    <string>创建课程</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
